tests/nosetests/pyanaconda_tests/modules/storage/platform_test.py

```diff
@@ -73,12 +73,11 @@ def _check_constraints(self, descriptions, constraints, error_message):
             "raid_levels": [],
             "raid_metadata": [],
             "raid_member_types": [],
-            "descriptions": descriptions
         }
         all_constraints.update(constraints)
 
         platform = get_platform()
-        self.assertEqual(platform.boot_stage1_constraint_dict["descriptions"], descriptions)
+        self.assertEqual(platform.stage1_descriptions, descriptions)
```

Comment:
This will work only on English locale.

Reply:
We set the locale to `en_US.UTF-8` in the unit tests, so we can test translated strings.

Reply:
Good point.